fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccount.java

```diff
@@ -140,11 +140,11 @@
     @JoinColumn(name = "client_id", nullable = true)
     protected Client client;
 
-    @ManyToOne(optional = true)
+    @ManyToOne(optional = true, fetch=FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToOne(optional = true, fetch=FetchType.LAZY)  // see FINERACT-902
```

Reply:
@rrpawar96 did you want to add this?

Reply:
Yes

Reply:
@rrpawar96 I think you still forgot this? I'll finally merge it anyway :smile: 

Perhaps you want to follow-up and add it in a subsequent PR.